docker-codescene/start-codescene.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-set -x
-
-mkdir -p $CODESCENE_ANALYSIS_RESULTS_ROOT
-mkdir -p $CODESCENE_CLONED_REPOSITORIES_ROOT
-# don't quote "$JAVA_OPTIONS" because you wouldn't be able to use it for multiple properties/settings in docker-compose.yml
-java -XshowSettings:vm $JAVA_OPTIONS -XX:MaxRAMPercentage=75 -Duser.timezone="$CODESCENE_TIMEZONE" -jar "/opt/codescene/codescene-enterprise-edition.standalone.jar" | tee -a ${CODESCENE_DIR}/codescene.log
```

Comment:
This wasn't reflected in the README. E.g. `-XX:MaxRAMPercentage` is missing (there are only [outdated instructions for old version of JDK 8](https://github.com/empear-analytics/docker-codescene-nginx-self-signed-ssl#memory-settings))

Reply:
Not sure what you mean - should I delete the outdated instructions?
The MaxRAMPercentage setting is included in our docker image. I we should write anything about that setting we should do it in the docker hub readme.